python/podio_schema_evolution.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python
+
+from podio_config_reader import PodioConfigReader
```

Comment:
```suggestion
from podio.podio_config_reader import PodioConfigReader
```
fix, necessary after #343